main/admin.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
-from .models import UserInformation, Job,AppliedJob
+from .models import *
```

Comment:
"*" might cause circular dependencies in some cases, it's better to use from .models import (
UserInformation, 
Job,AppliedJob
)